src/RageFileBasic.cpp

```diff
@@ -84,145 +84,145 @@ int RageFileObj::Seek( int iOffset )
 	 * flush before seeking to do proper error checking. */
 	ResetReadBuf();
 
-	int iPos = SeekInternal( iOffset );
-	if( iPos != -1 )
-	        m_iFilePos = iPos;
+	int iPos = SeekInternal(iOffset);
+	if (iPos != -1)
+		m_iFilePos = iPos;
 	return iPos;
 }
 
-int RageFileObj::Seek( int offset, int whence )
+int RageFileObj::Seek(int offset, int whence)
 {
-	switch( whence )
+	int iOffset = static_cast<int>(offset);
```

Comment:
Why do we need to cast an int to an int?? Seems like this local variable isn't needed at all here.

Reply:
Ahh yeah tbh not sure why I chose to do this.